doc/GUIDE.md

```diff
@@ -1574,13 +1580,36 @@ Selected resolver: lts-3.2
 Wrote project config to: /home/michael/my-yesod-project/stack.yaml
 ```
 
-Alternatively you can use your own templates by specifying the path:
+The default `stack-templates` repository is on [Github](https://github.com),
```

Comment:
How about adding a link directly to the repo?

Reply:
Sure, will do. There's also a direct link to the repo later in the section.